ts-tests/contracts/NetworkUpgrade.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity >=0.8.2 <0.9.0;
+
+contract NetworkUpgrade {
+
+    uint256 number;
+    string message;
+    uint256[] testArray;
+    
```

Comment:
We missed a lot of data structures:

Different types of arrays
Structs 
Key-value pairs
Enum
bytes
address

For example: 

```
struct Voter { // Struct
uint weight1, weight2;
bool voted;
address delegate1, delegate2;
string name;
uint vote1, vote2;
} }

//Array with a fixed length of 2 elements, containing integers :
uint[2] fixedArray;

//Another fixed Array, can contain 5 strings :
string[5] stringArray;

enum week_days
    {
      Monday,
      Tuesday,
      Wednesday,
      Thursday,
      Friday,
      Saturday,
      Sunday
     } 

contract Bytes_Array_Example{
    bytes1 one_character = "a"; // 1 character (✅)
    bytes2 two_characters = "ab"; // 2 characters (✅)
    bytes3 three_characters = "abc"; // 3 characters (✅)
    bytes4 four_characters = "abcd"; // 4 characters (✅)
    bytes5 five_characters = "abcde"; // 5 characters (✅)
    bytes32 thrity_two_characters = "abcdefghijklmnopqrstuvwxyz123456"; // 32 characters (✅)
}

 address user_address = 0x0000000000000000000000000000000000000000; // (✅)

```

Reply:
I tested only a handful of data structures - assuming if these are working, the rest should not be broken too. 

Is it required to test for each and every data structures?
In that case I would need to update the ink contracts too for the ink-test-cases